content/zh-cn/docs/tasks/extend-kubernetes/setup-konnectivity.md

```diff
@@ -35,7 +35,7 @@ The following steps require an egress configuration, for example:
 
 接下来的步骤需要出口配置，比如：
 
-{{< codenew file="admin/konnectivity/egress-selector-configuration.yaml" >}}
+{{% code file="admin/konnectivity/egress-selector-configuration.yaml" %}}
```

Comment:
/hold

Wait for the outcome of the discussion https://github.com/kubernetes/website/issues/42203#issuecomment-1691439368.